builtin/logical/aws/path_config_root.go

```diff
@@ -259,22 +240,22 @@ func (b *backend) pathConfigRootWrite(ctx context.Context, req *logical.Request,
 	return nil, nil
 }
 
-func getConfigFromStorage(ctx context.Context, req *logical.Request) (*rootConfig, bool, error) {
+func getConfigFromStorage(ctx context.Context, req *logical.Request) (*rootConfig, error) {
```

Comment:
I'm trying to decide if 1) I object to this lint, 2) we should return an error for an empty entry on line 249, or 3) we should add back the boolean.

Reply:
I think the bool wasn't necessary because any time there's an error return it's due to a real error. But when there's not an error, either a config was found or there was not. A nil config is valid as it just means there isn't one yet, such as reading a backend config before the engine is configured.